src/main/java/dev/mlml/command/Context.java

```diff
@@ -41,7 +44,7 @@ public Context(Message message) {
         this.member = message.getMember();
         this.gChannel = message.getGuildChannel();
         this.channel = message.getChannel();
-
+        this.guild = message.getGuild();
```

Comment:
Maybe refactor this to the rest of the codebase

Reply:
There's no other uses of this, thought `Message.getGuildId()` is used elsewhere.